recipe/meta.yaml

```diff
@@ -24,21 +22,20 @@ requirements:
     - wheel
   run:
     - python
-    - packaging >=17.1
+    - packaging >=20.0s
     - numpy >=1.17.2
     - pyyaml >=5.4
-    - pytorch >=1.11.0
+    - pytorch >=2.0.0
     - tqdm >=4.57.0
-    - fsspec >2021.06.1
+    - fsspec >2022.5.0
     - torchmetrics >=0.7.0
-    - typing-extensions >=4.0.0
-    - lightning-utilities >=0.7.0
+    - typing-extensions >=4.4.0
+    - lightning-utilities >=0.8.0
```

Comment:
Maybe add a note in that we're not supporting lightning.app at the moment. You could also update [lightning-cloud](https://github.com/AnacondaRecipes/lightning-cloud-feedstock/blob/main/recipe/meta.yaml) and include the required deps, if you want, but I'll leave that up to you

Reply:
I left a comment about the source of the package's dependencies